File: RealEstate-Source-Code-v6.0/AdminPanel-Laravel10/app/Models/News.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class News extends Model
{
    protected $table = "news";
    protected $fillable = ['category_id', 'title', 'content', 'image_name'];

    public function category()
    {
        return $this->belongsTo(NewsCategory::class, 'category_id');
    }
}

File: RealEstate-Source-Code-v6.0/AdminPanel-NetCore/Entities/News.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RealEstate.Entities
{
    public partial class News
    {
        [Key]
        public int Id { get; set; }
        public int CategoryId { get; set; }
        [Required]
        [StringLength(250)]
        public string Title { get; set; }
        [Required]
        public string Content { get; set; }
        [Required]
        [StringLength(250)]
        public string ImageName { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime CreatedDate { get; set; }

        [ForeignKey(nameof(CategoryId))]
        [InverseProperty(nameof(NewsCategory.News))]
        public virtual NewsCategory Category { get; set; }
    }
}
